src/finch/dtypes.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+from .julia import jl
+
+
+_dtype_pairs = [
+    (np.int_, jl.Int),
+    (np.int8, jl.Int8),
+    (np.int16, jl.Int16),
+    (np.int32, jl.Int32),
+    (np.int64, jl.Int64),
+    (np.uint, jl.UInt),
+    (np.uint8, jl.UInt8),
+    (np.uint16, jl.UInt16),
+    (np.uint32, jl.UInt32),
+    (np.uint64, jl.UInt64),
+    (np.float16, jl.Float16),
+    (np.float32, jl.Float32),
+    (np.float64, jl.Float64),
+    (np.complex64, jl.ComplexF32),
+    (np.complex128, jl.ComplexF64),
+    (np.bool_, jl.Bool),
+]
+
+dtype_py_to_jl = {pair[0]: pair[1] for pair in _dtype_pairs}
+
+dtype_jl_to_py = {pair[1]: pair[0] for pair in _dtype_pairs}
```

Comment:
We don't really need this -- the Array API standard stipulates that we can use any dtype objects, not necessarily NumPy ones. I think something as simple as

```python
int_ = jl.Int
int8 = jl.Int8
...
```

might suffice.

Reply:
That's right - done!